Lab2/server/nested_templates/userinterface.yaml

```diff
@@ -5,14 +5,23 @@ AWSTemplateFormatVersion: "2010-09-09"
 Transform: AWS::Serverless-2016-10-31
 Description: >
   Template to deploy cloudfront and s3 bucket for UI code
+Parameters:
+  EventEngineParameter:
```

Comment:
I would make the param more generic and call as IsCloudFrontPreProvisioned or something like that

Reply:
Assuming you just want me to change this parameter name only in this userinterface.yaml file not in shared-template.yaml/template.yaml. Please let me know otherwise.